### DISCUSSION TITLE
Unexpected token '<', " <!DOCTYPE "... is not valid JSON

### DISCUSSION THREAD

**Discussion by edwardk1234** [lllyasviel/Fooocus#1464] [General]
When I try to run inpainting, I get this error message "Unexpected token '<', " <!DOCTYPE "... is not valid JSON"

Here are some screenshots. How do I solve this?
![Error 1](https://github.com/lllyasviel/Fooocus/assets/56983402/b9aef109-33f2-4cda-9752-9b14f6dc97ec)
![Error 2](https://github.com/lllyasviel/Fooocus/assets/56983402/9de9621b-6b9c-4976-a45f-60bb0c12b728)

**Comment by dyb323lu** [comment]
The session crashed, just try running it again

**Reply by mashb1t** [reply]
^ this happens when your WS disconnects or the session crashes. On mobile devices Gradio also outputs a warning that this may happen.

**Comment by maxim640** [comment]
Same here... :-(
